sdk/python/lib/pulumi/automation/_stack.py

```diff
@@ -606,6 +639,85 @@ def destroy(
             stdout=destroy_result.stdout, stderr=destroy_result.stderr, summary=summary
         )
 
+    def import_resources(
+        self,
+        message: Optional[str] = None,
+        resources: Optional[List[ImportResource]] = None,
+        nameTable: Optional[Dict[str, str]] = None,
+        protect: Optional[bool] = None,
+        generate_code: Optional[bool] = None,
+        converter: Optional[str] = None,
+        converter_args: Optional[List[str]] = None,
+        on_output: Optional[OnOutput] = None,
+        show_secrets: Optional[bool] = True,
+    ) -> ImportResult:
+        """
+        Imports resources into the stack.
+        
+        :param message: Message to associate with the import operation.
+        :param resources: The resources to import.
+        :param nameTable: \
+            The name table maps language names to parent and provider URNs. \
+            These names are used in the generated definitions, \
+            and should match the corresponding declarations \
+            in the source program. This table is required if any parents or providers are \
+            specified by the resources to import.
+        :param protect: Whether to protect the imported resources so that they are not deleted
+        :param generate_code: Whether to generate code for the imported resources
+        :param converter: The converter plugin to use for the import operation
+        :param converter_args: Additional arguments to pass to the converter plugin
+        :param on_output: A function to process the stdout stream.
+        :param show_secrets: Include config secrets in the ImportResult summary.
+        """
+        args = ["import", "--yes", "--skip-preview"]
+        if message is not None:
+            args.extend(["--message", message])
+
+        tempDir = ""
+        try:
+            tempDir = tempfile.mkdtemp(prefix="pulumi-import-")
+            if resources is not None:
+                importFilePath = os.path.join(tempDir, "import.json")
```

Comment:
Again, let's stick to snake_case in Python.

Reply:
Fixed ✅